apps/meeting/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "chime-sdk-meeting-demo",
+  "version": "0.1.0",
+  "dependencies": {
+    "@types/react": "^16.9.46",
+    "@types/react-dom": "^16.9.5",
+    "@types/styled-components": "^4.4.2",
+    "@types/webpack-env": "^1.15.1",
+    "@typescript-eslint/eslint-plugin": "^2.19.2",
+    "@typescript-eslint/parser": "^2.19.2",
+    "amazon-chime-sdk-component-library-react": "^2.6.0",
+    "amazon-chime-sdk-js": "^2.11.0",
+    "aws-sdk": "^2.731.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router-dom": "^5.1.2",
+    "styled-components": "^5.0.0",
+    "styled-system": "^5.1.5",
+    "throttle-debounce": "^2.3.0"
+  },
+  "scripts": {
+    "lint": "eslint src --ext .js,.jsx --fix",
+    "build": "webpack --config ./webpack.config.js --mode=production --devtool=false",
+    "start:client": "npm run build && webpack-dev-server",
+    "start:backend": "node server.js",
+    "start": "concurrently \"npm run start:client\" \"npm run start:backend\""
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.1",
+    "@babel/plugin-transform-regenerator": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@babel/runtime": "^7.11.2",
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
+    "@types/react-router-dom": "^5.1.3",
+    "@types/styled-system": "^5.1.9",
+    "@types/throttle-debounce": "^2.1.0",
+    "babel-eslint": "^9.0.0",
+    "babel-loader": "^8.0.4",
+    "concurrently": "5.2.0",
+    "css-loader": "^5.2.6",
+    "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-config-wesbos": "^0.0.19",
+    "eslint-plugin-html": "^5.0.5",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.20.5",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "html-webpack-inline-source-plugin": "0.0.10",
+    "html-webpack-plugin": "^3.2.0",
+    "prettier": "^1.19.1",
+    "react-is": "^17.0.0",
```

Comment:
Should this be dev dependency or dependency?

Reply:
If you mean the `react-is`, then it should be in dev dependency, because I tried to put it in the dependency and the demo failed...
For other three, I am not sure, I just keep it as is

Reply:
I just meant 'react-is'.